src/lib/core/style-layer/circular-progress/_core.scss

```diff
@@ -0,0 +1,126 @@
+@use './variables';
+@use '../../../theme/theme-values';
+
+/// Configurable module-level variables
+$indeterminate-animation: linear infinite linear-rotate !default;
+$indeterminate-circle-animation: expand-arc !default;
+$determinate-animation: infinite both rotate-arc !default;
+
+///
+/// Initializes private configuration state CSS variables
+/// 
+@mixin configuration {
+  --_size: var(--#{variables.$prefix}-size, #{variables.$size});
+  --_spacing: var(--#{variables.$prefix}-padding, #{variables.$spacing});
+  --_track-width: var(--#{variables.$prefix}-indicator-width, #{variables.$track-width});
+  --_track-color: var(--#{variables.$prefix}-indicator-color, var(--forge-theme-tertiary, #{theme-values.$tertiary}));
+  --_track-background: var(--#{variables.$prefix}-indicator-background-color, var(--mdc-theme-text-disabled-on-background, #{theme-values.$text-disabled-on-background}));
```

Comment:
This might be a separate discussion, but is the goal to eventually have all `--forge-` variables and no `--mdc-`, or are there still some central theme ones we'll always use?